packages/api-explorer/src/ResponseExample.jsx

```diff
@@ -259,7 +280,7 @@ class ResponseExample extends React.Component {
           </span>
         )}
 
-        {(examples.length === 0 || (!hasExamples && result === null)) && (
+        {(examples.length === 0 || !result === null) && (
```

Comment:
`!result === null` will never be true, I'm guessing this is supposed to be `result === null`. FWIW I also prefer `result == null` cause that's inclusive of both (and only) null & undefined. Lodash's [isNil](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11974) is just that.

Reply:
I'm going to remove this `result` check and the prop from this component completely since `examples.length` is always going to take precedence over this block now that we're handling empty responses.